devicetypes/smartthings/smartsense-moisture-sensor.src/smartsense-moisture-sensor.groovy

```diff
@@ -244,11 +266,11 @@ private Map getTemperatureResult(value) {
 		def v = value as int
 		value = v + offset
 	}
-    def descriptionText
-    if ( temperatureScale == 'C' )
-    	descriptionText = '{{ device.displayName }} was {{ value }}°C'
-    else
-    	descriptionText = '{{ device.displayName }} was {{ value }}°F'
+	def descriptionText
+	if ( temperatureScale == 'C' )
+		descriptionText = '{{ device.displayName }} was {{ value }}°C'
```

Comment:
This is how internationalization was implemented. I just changed the spaces to tabs to be consistent with our coding style.